src/plugins/news/services/ArticleSubscriber.js

```diff
@@ -228,6 +231,11 @@ export default class ArticleSubscriber extends EventSubscriber {
       node.addToList('related_article_refs', data.relatedArticleRefs);
     }
 
```

Comment:
@albertgcsula there should be a HasRelatedTeasersSubscriber that does this for all instead of having code in each node type subscriber

Reply:
@gdbrown okay, created.